packages/components/src/scroll-to-top/scroll-to-top-on-mount.tsx

```diff
@@ -0,0 +1,10 @@
+import { useEffect } from 'react';
+
+export function ScrollToTopOnMount() {
```

Comment:
It's generally a good practice for every new `@automattic` component to have:

* A `docs/example`
* A README
* Tests
* Representation in `devdocs` and/or `storybook`.

Reply:
Yes, I like to use `storybook` because it has the fast refresh built in, and it's isolated from the rest of the UI, which makes your component more reusable. If I keep this component, I'll add the necessary documentation.